src/element.js

```diff
@@ -36,7 +36,21 @@ function findElement(sendSessionCommand, strategy, selector) {
             getText: getText.bind(null, sendElementCommand),
             getCss: getCss.bind(null, sendElementCommand)
         };
-    });
+    }).catch(() => undefined);
+}
+
+async function findElement(sendSessionCommand, strategy, selector, maxWaitTime = 0) {
```

Comment:
I would like to keep the findElement as is and add a different function named `elementPresent`. Also I think it should be implemented using https://www.w3.org/TR/webdriver/#execute-async-script. And polling should be done on the client side for performance and network traffic reasons.

What do you think?